README.md

```diff
@@ -319,7 +319,7 @@ Benchmark comparison test [details data](https://gist.github.com/membphis/137db9
 
 A wide variety of companies and organizations use APISIX for research, production and commercial product, including:
 
-<img src="https://raw.githubusercontent.com/api7/website-of-API7/master/user-wall.jpg" width="900" height="500" />
+<img src="https://user-images.githubusercontent.com/40708551/109468444-95aeb200-7a92-11eb-9765-c25522d84e41.png" width="725" height="1000" />
```

Comment:
Hi, I noticed that there has a red title 🤔 we will not use it.

Also, we have 40+ users and there have 25 users in this picture, we need all users.

Reply:
for that i need to add 1 more logo page, will that be ok?

Reply:
will this file[1] help you? :)

[1] https://github.com/apache/apisix-website/blob/master/website/static/data/showcases.json#L1

Reply:
> will this file[1] help you? :)
> 
> [1] https://github.com/apache/apisix-website/blob/master/website/static/data/showcases.json#L1

> absolutely, thanks